src/checkers/inference/InferenceOptions.java

```diff
@@ -69,6 +71,9 @@
     @Option("[InferenceSolver] the fully-qualified name of the solver to use on constraints; overrides --typesystem.")
```

Comment:
Should the "overrides --typesystem" be "overrides --solver" or something else? Why would checker and solver influence the type system?